### ISSUE TITLE
Create a Website for this Repo

### ISSUE THREAD

**Issue by xdvrx1** [charignon/github-review#47]
Hi there. 

Is there a website for this repo? 

Because if you don't have, well, this repo can simply be turned into a website right away. Others will discover this project in that website.

Steps:
1. Go to Settings and look for `GitHub Pages`, scroll down. That's almost at the bottom.

2. You will see there: `Branch:none`, so you should change that to `master` because you have a `README.md` file in the master repo. This will be your page. Click `Save` first.

3. Then click `Choose a theme`, you select a predefined theme of your site.

4. Visit your site now! The URL will be `https://charignon.github.io/github-review`. 

If you were amazed by that, simply read the documentation about GitHub Pages.

**Comment by charignon** [comment]
Hello! Thanks for your interest, the suggestion and the detailed steps :)

There is none, the closer I have of a website for it is the article I wrote originally https://blog.laurentcharignon.com/post/code-review-in-emacs/

I feel like the emacs community discovers packages through blog articles and reddit. This package is mentioned by several blogs including Sacha Chua's and was mentioned on reddit. If you feel like a site would help for discoverability within the community, please upvote this comment and I will do it.

**Comment by xdvrx1** [comment]
Yup, it really helps because those big companies like Twitter, their Bootstrap project, they have the website solely dedicated for the project and its documentation. Thanks a lot for the response.
